### PR TITLE
Spelling aclk

### PR DESCRIPTION
**Pull request by jsoref**
##### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

##### Component Name

aclk

##### Test Plan

The misspellings have been reported at https://github.com/jsoref/netdata/commit/33c5b20ed9511dac0a23ee916fa247cc2f388304#commitcomment-45444734

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/netdata/commit/e2737c090e4b1d7b1bce405b9f702bcf526852a3

##### Additional Information

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

This is split from #10521 per https://github.com/netdata/netdata/pull/10521#issuecomment-813622899

### CODE REVIEW

**Review by vlvkobal — DISMISSED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata-all
           sudo apt-get install -y libjson-c-dev libipmimonitoring-dev libcups2-dev libsnappy-dev \
                                   libprotobuf-dev libprotoc-dev libssl-dev protobuf-compiler \
-                                  libnetfilter-acct-dev libmongoc-dev  libcmocka-dev
+                                  libnetfilter-acct-dev libmongoc-dev  libcmocka-dev libzstd-dev
```
Comment (underhood) [inline .github/workflows/tests.yml:51]: should this be in PR that fixes spelling?

Comment (ilyam8) [reply .github/workflows/tests.yml:51]: @jsoref added this to every his PR to fix cmocka units tests. If i understand it correctly, no matter which one we merge first, it won't result in conflict merges.

But i agree that this is not related change and should be a separate PR.

Comment (underhood) [reply .github/workflows/tests.yml:51]: I would be strongly in favor of separate PR - fix cmoka unit tests, then remove it from partial PRs. Searching for such change in history if it is in or named "spelling fixes"... i can see future somebody being happy :D

Comment (ilyam8) [reply .github/workflows/tests.yml:51]: @jsoref from what i've noticed there are 2 unrelated to spelling changes:
 - fix cmocka unit tests
 - upate reviewdog es-lint action

I understand that moving those changes out from the spelling PRs means a lot of work, but that would be cleaner. That is a very good point about changes history from @underhood

Comment (jsoref) [reply .github/workflows/tests.yml:51]: No, but it's hard to prove that my changes are safe if the tests don't run.

And spelling changes can easily break tests, so the better the coverage, the happier everyone is.

I'm happy to split the fix out into a distinct PR (there's already a distinct issue for it).

Comment (ilyam8) [reply .github/workflows/tests.yml:51]: We can merge cmocka units tests PR fix right away and then you rebase spelling PRs on the master branch

Comment (jsoref) [reply .github/workflows/tests.yml:51]: @ilyam8: it's actually pretty trivial for me to do these things _en mas_, so don't worry about the amount of work. As long as I don't have to manually filter out comments vs code, I'm happy.

These additional commits were me trying to convince the CI that my changes were good and green. The cmocka unit test fix appears to have worked in that case. The es-lint one was a total failure.

- Amongst other things, it doesn't actually do what I expected, I should know better, but I don't actively pay attention to how people use action version tags -- my own action doesn't have a `v1` yet, just static releases, and the latest version of the eslint action just seems to be broken here ... but, that's not a problem I need to solve if people here don't need me to solve it -- and I shouldn't need to solve it for these fixes.)

Comment (underhood) [reply .github/workflows/tests.yml:51]: Thanks a lot for the cmocka CI fix.... that thing has been red for what seems like forever so very happy about that contribution :)

Comment (ilyam8) [reply .github/workflows/tests.yml:51]: > As long as I don't have to manually filter out comments vs code, I'm happy.

😄

Comment (jsoref) [reply .github/workflows/tests.yml]: For the curious, I split the cmocka fix out to #10925

**Review by underhood — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**